oq_output/disaggregation_converter.py

```diff
@@ -140,7 +142,10 @@ def save_disagg_to_csv(nrml_disaggregation, output_dir, plot):
         if len(axis) == 1:
             values = numpy.array([axis[0], matrix.flatten()]).T
         else:
-            grids = numpy.meshgrid(*axis, indexing='ij')
+            try:
```

Comment:
This seems very defensive. There really isn't a case in which the code can execute as far as this point and utils.meshgrid function not work - so theoretically it will never be possible to reach the nested except.